### PR TITLE
update graph security doc: required permissions

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content-docs/issues/296

## Description
Update to the actual required permissions.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 1]
```diff
@@ -10,12 +10,11 @@
...
-<li>SecurityEvents.ReadWrite.All - Application</li>
-<li>User.Read.All - Application</li>
-<li>User.Read - Delegated</li>
-<li>User.ReadWrite.All - Application</li>
-<li>Directory.Read.All - Delegated</li>
-<li>Directory.ReadWrite.All - Application</li>
+<ul>
+<li>SecurityEvents.Read.All - Application</li>
+<li>SecurityEvents.ReadWrite.All - Application (required for updating alerts with the command: <code>msg-update-alert</code>)</li>
+<li>User.Read.All - Application (required for the commands: <code>msg-get-user</code> and <code>msg-get-users</code>)</li>
```
Comment (Itay4) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:16]: i think we can remove that one as the commands will be deprecated in https://github.com/demisto/content/pull/9033

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md [hunk 2]
```diff
@@ -10,12 +10,11 @@
...
 
 <h3>Required Permissions</h3>
-<li>SecurityEvents.ReadWrite.All - Application</li>
-<li>User.Read.All - Application</li>
-<li>User.Read - Delegated</li>
-<li>User.ReadWrite.All - Application</li>
-<li>Directory.Read.All - Delegated</li>
-<li>Directory.ReadWrite.All - Application</li>
+<ul>
+<li>SecurityEvents.Read.All - Application</li>
```
Comment (Itay4) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:14]:
```suggestion
<li>SecurityEvents.Read.All - Application (required for the commands: <code>msg-search-alerts</code> and <code>msg-get-alert-details</code>)</li>
```

**Review by Itay4 — APPROVED** [review]

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md (hunk 1 above)
Comment (glicht) [reply Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md:16]: Will add a note that these commands are deprecated.

**Comment by glicht** [comment]
force merge. unrelated failures.

**Pull request merged**